### ISSUE TITLE
broken install

### ISSUE THREAD

**Issue by hakavlad** [rfjakob/earlyoom#77]
git clone https://github.com/rfjakob/earlyoom.git
Клонирование в «earlyoom»…
remote: Counting objects: 568, done.
remote: Compressing objects: 100% (85/85), done.
remote: Total 568 (delta 69), reused 101 (delta 43), pack-reused 439
Получение объектов: 100% (568/568), 136.47 KiB | 0 bytes/s, готово.
Определение изменений: 100% (326/326), готово.
$ cd earlyoom
$ make
cc  -Wall -Wextra -DVERSION=\"v1.1-12-g647107d\" -g -fstack-protector-all -std=gnu99 -o earlyoom meminfo.c kill.c main.c sanitize.c
pandoc MANPAGE.md -s -t man > earlyoom.1
/bin/sh: 1: pandoc: not found
Makefile:22: ошибка выполнения рецепта для цели «earlyoom.1»
make: *** [earlyoom.1] Error 127

**Comment by hakavlad** [comment]
@rfjakob 
make
cc  -Wall -Wextra -DVERSION=\"v1.1-25-gc350b04\" -g -fstack-protector-all -std=gnu99 -o earlyoom meminfo.c kill.c main.c sanitize.c
pandoc is not installed, **skipping earlyoom.1 manpage generation**
Is it OK?

**Comment by hakavlad** [comment]
@rfjakob 
sudo make install
[sudo] пароль для user: 
sed "s|:TARGET:|/usr/local/bin|g;s|:SYSCONFDIR:|/etc|g" earlyoom.service.in > earlyoom.service
install -d /usr/local/bin/
install -m 755 earlyoom /usr/local/bin/
pandoc is not installed, **skipping earlyoom.1 manpage generation**
gzip -k earlyoom.1
**gzip: earlyoom.1: No such file or directory**
Makefile:58: ошибка выполнения рецепта для цели «earlyoom.1.gz»                                                                                       
make: *** [earlyoom.1.gz] Ошибка 1 

Seems like something went wrong.

**Comment by hakavlad** [comment]
New man not installed, it is a bug.

**Comment by rfjakob** [comment]
If you want to install, you need pandoc

**Comment by hakavlad** [comment]
it not mentioned in readme

**Comment by hakavlad** [comment]
and pandoc has a huge size:

sudo apt install pandoc
[sudo] пароль для user: 
Чтение списков пакетов… Готово
Построение дерева зависимостей       
Чтение информации о состоянии… Готово
Будут установлены следующие дополнительные пакеты:
  libluajit-5.1-2 libluajit-5.1-common pandoc-data
Предлагаемые пакеты:
  texlive-latex-recommended texlive-xetex texlive-luatex pandoc-citeproc
  texlive-latex-extra wkhtmltopdf
НОВЫЕ пакеты, которые будут установлены:
  libluajit-5.1-2 libluajit-5.1-common pandoc pandoc-data
обновлено 0, установлено 4 новых пакетов, для удаления отмечено 0 пакетов, и 0 пакетов не обновлено.
Необходимо скачать 9 612 kБ архивов.
После данной операции, объём занятого дискового пространства возрастёт на **72,2 MB**.
Хотите продолжить? [Д/н]

**Comment by rfjakob** [comment]
Yes, that's why it is optional for compiling. But it's still mandatory for installing, as this needs the man page.

**Comment by hakavlad** [comment]
I think that users should be able to install the man page without even having pandoc

**Comment by rfjakob** [comment]
They can, by using a precompiled package. If you want to compile and install yourself, you need the dependencies (gcc and pandoc).

**Comment by hakavlad** [comment]
ok

**Comment by rfjakob** [comment]
Ok I just got the second ticket about this: https://github.com/rfjakob/earlyoom/issues/85

Now "make install" also skips the manpage if pandoc is missing: https://github.com/rfjakob/earlyoom/commit/c58b5813e0cb666815b7a836e98c49df3974ce82 . Obviously you won't have a man page then.
